docker/zcash-params/Dockerfile

```diff
@@ -28,9 +28,15 @@ ENV CARGO_HOME /app/.cargo/
 # Build dependencies - this is the caching Docker layer!
 RUN cargo chef cook --release --features enable-sentry --recipe-path recipe.json
 
-ARG RUST_BACKTRACE=1
+ARG RUST_BACKTRACE=0
 ENV RUST_BACKTRACE ${RUST_BACKTRACE}
 
+ARG RUST_LIB_BACKTRACE=0
+ENV RUST_LIB_BACKTRACE ${RUST_LIB_BACKTRACE}
+
+ARG COLORBT_SHOW_HIDDEN=0
+ENV COLORBT_SHOW_HIDDEN ${COLORBT_SHOW_HIDDEN}
```

Comment:
Optional: These env var commands don't quite match `docker/Dockerfile`, can we use the same style in both places?

Reply:
I'll create a PR to standardize all Dockerfiles and to document a few steps.